src/module/vue/components/progress/progress-list-item.vue

```diff
@@ -0,0 +1,238 @@
+        v-if="item.data.hasTrack"
+        icon="caret-right"
+        @click="advance"
+        :tooltip="$t('IRONSWORN.MarkProgress')"
+      />
+      <BtnRollprogress
+        v-if="item.data.hasTrack"
+        :item="item"
+        :tooltip="$t('IRONSWORN.ProgressRoll')"
+        class="block"
+      />
+      <BtnFaicon
+        v-if="showStar"
+        class="star-progress block"
+        icon="star"
+        :solid="item.data.starred"
+        :tooltip="$t('IRONSWORN.StarProgress')"
+        data-tooltip-direction="RIGHT"
+        @click="toggleStar"
+      />
+    </section>
+  </article>
+</template>
+
+<style lang="less" scoped>
+@progress_widget_spacing: 6px;
+.progress-list-item {
+  padding: (@progress_widget_spacing / 2) (@progress_widget_spacing / 2)
+    @progress_widget_spacing;
+  display: grid;
+  grid-template-columns: max-content max-content 1fr max-content;
+  grid-template-rows: max-content max-content 1fr;
+  gap: @progress_widget_spacing;
+  .progress-img {
+    grid-column: 2;
+    grid-row: 1 / span 2;
+    margin: 0;
+  }
+  .progress-rank-pips {
+    grid-column: 3 / span 2;
+    grid-row: 1;
+  }
+  .progress-title {
+    grid-column: 3;
+    grid-row: 2;
+    margin: 0;
+    line-height: 1;
+    height: max-content;
+  }
+  .progress-subtitle {
+    grid-column: 1;
+    grid-row: 1 / span 3;
+    padding: 0;
+    line-height: 1;
+    width: max-content;
+    padding: 0;
+    margin: 0;
+    font-weight: normal;
+  }
+  .progress-widgets {
+    grid-column: 2 / span 3;
+    grid-row: 3;
+    gap: 2px;
+    .progress-clock {
+      flex-basis: 50px;
+    }
+  }
+  .progress-controls {
+    display: grid;
+    grid-column: 4;
+    grid-row: 1 / span 2;
+    grid-auto-flow: column;
+    > * {
+      aspect-ratio: 1;
+      grid-row: 1;
+    }
+    .star-progress {
+      grid-row: 2;
+    }
+  }
+}
+</style>
+
+<script lang="ts" setup>
+import { capitalize, computed, inject, provide } from 'vue'
+import { $ActorKey } from '../../provisions'
+import Clock from '../clock.vue'
+import Track from './track.vue'
+import BtnRollprogress from '../buttons/btn-rollprogress.vue'
+import BtnFaicon from '../buttons/btn-faicon.vue'
+import RankPips from '../rank-pips/rank-pips.vue'
+import DocumentImg from '../document-img.vue'
+import { RANKS } from '../../../constants.js'
+import ProgressTrack from './progress-track.vue'
+
+const props = defineProps<{
+  item: any
+  actor: any
+  showStar?: boolean
+  /**
+   * When true, renders the progress bar for more compact display.
+   */
+  compactProgress?: boolean
+}>()
+
+const $actor = inject($ActorKey)
+provide(
+  'actor',
+  computed(() => props.actor)
+)
+
+// ProgressListItem is embedded in a sheet but the sheet usually doesn't provide it -- in other words, this
+// So, get the id of the nonreactive item
+const progressId = computed<string>(() => props.item.id ?? props.item._id)
+// Then use it in a getter, $item
+const $item = computed(() => $actor?.items.get(progressId.value))
```

Comment:
I was initially confused by seeing `$item?.value?.[...]` below, and had to scroll up to see why. Everywhere else this notation indicates that this is injected using `$ItemKey`, and is in fact an `IronswornItem`, but here you're using that name for a ref. Let's keep the `$`'s meaning, and rename this to something like `actorItem`.

Better yet, it doesn't need to be computed, because neither actor nor item will be changing while this component is mounted:

```ts
const actorItem = $actor?.items.get(progressId.value)
```

Then you can drop the `?.value` everywhere.